src/components/DateRangePickerInputController.jsx

```diff
@@ -28,6 +28,8 @@ import {
 } from '../constants';
 
 const propTypes = forbidExtraProps({
+  children: PropTypes.node,
```

Comment:
do we need to allow any kind of children here?

Reply:
Would be good to restrict

Reply:
@TaeKimJR added this stuff, so pinging them to get thoughts on this approach and whether we should change it :)

Reply:
Restricting makes sense!

Reply:
I did some restricting and it looks like we use this

Reply:
what things did it break?

it's mainly that, we can loosen easily later, but restricting later is a hassle, so it's always nicer to restrict as much as possible at the start

Reply:
Can you clarify what it broke? It's a new prop so I feel like we should be able to control the type of what's passed in

Reply:
Seems we're having the same conversation in two places 😺 See here: https://github.com/airbnb/react-dates/pull/1499/files#r245158005